carapace-server/src/main/java/org/carapaceproxy/core/ProxyRequestsManager.java

```diff
@@ -522,6 +532,11 @@ private Publisher<Void> serveFromCache(ProxyRequest request, ContentsCache.Conte
             headers.add(HttpHeaderNames.EXPIRES, HttpUtils.formatDateHeader(new java.util.Date(content.getExpiresTs())));
             request.setResponseHeaders(headers);
             addCustomResponseHeaders(request, request.getAction().customHeaders);
+            //If the request is http 1.0 we make sure to send without chunked
+            if (parent.getCurrentConfiguration().isHttp10BackwardCompatibilityEnabled() &&
+                    request.getRequest().version() == HttpVersion.HTTP_1_0) {
```

Comment:
@pv3nturi We need to introduce this part, becuse Flux.fromIterable return a new Flux and publisher in this case can emit more than 1 element. 

However Mono.from ensure that publisher will emit 0 or 1 item.

Reply:
Good catch!